Brewfile

```diff
@@ -2,6 +2,5 @@ brew 'postgresql@14'
 brew 'redis'
 brew 'node@22'
 brew 'yarn'
-brew 'openssl@1.1'
```

Comment:
Some of our deployed environments still use openssl 1.1, so I think it'd be preferable to keep it as-is for consistency.

Reply:
Is there a way to resolve the fact that this prevents engs from deploying idp on their local environments?

Reply:
What's the dependency for the environments that still use openssl 1.1? Login.gov is very good about keeping important things like Ruby versions up to date, which is why this change felt safe to me.